res/TensorFlowPythonExamples/examples/bidirectional_sequence_LSTM/__init__.py

```diff
@@ -0,0 +1,6 @@
+import tensorflow as tf
```

Comment:
Maybe this file should be removed?

Reply:
Yes, It should be, I have removed it in the latest. Thanks for spotting.